src/common/view-definitions/version-2/view-definition-converter-2.mocha.ts

```diff
@@ -17,14 +17,15 @@
 import { expect } from "chai";
 import { MANIFESTS } from "../../manifests";
 import { DataCubeFixtures } from "../../models/data-cube/data-cube.fixtures";
+import { Duration } from "chronoshift";
 import { TimeFilterPeriod } from "../../models/filter-clause/filter-clause";
 import { FilterClauseFixtures } from "../../models/filter-clause/filter-clause.fixtures";
 import { ViewDefinition2 } from "./view-definition-2";
 import { ViewDefinitionConverter2 } from "./view-definition-converter-2";
 
 describe("ViewDefinitionConverter2", () => {
 
-  const totalsWithTimeBucket: ViewDefinition2 = {
+  const totalsWithCurrentTimeBucket: ViewDefinition2 = {
```

Comment:
It is hard to see what is changing in the definitions between the next tree tests. You could extract a function so you only need to provide the expression and the unchanging parts are part of the function.

E.g.: 

```
const totalsWithCurrentTimeBucket = viewDefintionV2WithExpression({
        op: "timeBucket",
        operand: {
          op: "ref",
          name: "n"
        },
        duration: "P1D"
      })
```

Sorry indentation is hard on github 😸